packages/web/src/scss/components/Stack/README.md

```diff
@@ -31,4 +37,85 @@ Usage with a list:
 </ul>
 ```
 
-👉 Vertical margin of items inside Stack is reset to zero.
+## Variants
+
+⚠ Since spacing is implemented using the CSS `padding` property to create dividers using the `border` property,
```

Comment:
This is not correct for current default behaviour, right? This should be in the v1 section.

Reply:
I've edit the paragraph, now it's connect with `Stack--hasSpacing` class, so it could stay in this section.